libcontainer/cgroups/stats.go

```diff
@@ -49,6 +49,11 @@ type MemoryStats struct {
 	Stats       map[string]uint64 `json:"stats,omitempty"`
 }
 
+type PidsStats struct {
+	// current counter usage
+	Current uint64 `json:"current,omitempty"`
```

Comment:
I prefer it matches the name of the sysfs file. But it doesn't really matter I guess.